docs/user_guide/transformation/ArcsinTransformer.rst

```diff
@@ -0,0 +1,85 @@
+.. _arcsin:
+
+.. currentmodule:: feature_engine.transformation
+
+ArcsinTransformer
+=================
+
+The :class:`ArcsinTransformer()` applies the arcsin transformation to
+numerical variables.
+
+The :class:`ArcsinTransformer()` only works with numerical variables with values between 0 and +1. If the variable contains a value outside of this range, the transformer will raise an error.
+
+Let's load the breast cancer dataset and  separate it into train and test sets (more details about the dataset :ref:`here <datasets>`).
+
+.. code:: python
+
+    import numpy as np
+    import pandas as pd
+    import matplotlib.pyplot as plt
+    from sklearn.model_selection import train_test_split
+    from sklearn.datasets import load_breast_cancer
+
+    from feature_engine import transformation as vt
+      
+    #Load dataset
+    breast_cancer = load_breast_cancer()
+    X = pd.DataFrame(breast_cancer.data, columns=breast_cancer.feature_names)
+    y = breast_cancer.target
+
+  # Separate into train and test sets
+  X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=0)
```

Comment:
these 2 lines don't have the same indentation as above, then it shows funny in the docs.

Did you try building the docs locally? there are some guidelines here in case you are curious:
https://feature-engine.readthedocs.io/en/latest/contribute/contribute_docs.html#build-the-documentation

Reply:
I did build the docs locally and there were no issue with the building of the ArcsinTransformer.rst. At least I do not remember any error message, I do not think that I have checked the actual result of the build. Anyway I changed the indentation.